### PR TITLE
fix(types): add missing `goTo` types

### LINKED ISSUE #9648: [Bug Report] No typings for import goTo function
### Environment
**Vuetify Version:** 2.1.9
**Vue Version:** 2.6.10
**Browsers:** Safari 13.0.3, Google Chrome, Mozilla Firefox, Safari, Microsoft Edge, Internet Explorer, Opera, Other
**OS:** Mac OS 10.14.6

### Steps to reproduce
Just try to import goTo function from typescript:

`import goTo from "vuetify/lib/services/goto";`

### Expected Behavior
successful import

### Actual Behavior
TS Error:
`Could not find a declaration file for module 'vuetify/lib/services/goto'`

### Reproduction Link
<a href="https://codesandbox.io/s/zen-taussig-3dwph" target="_blank">https://codesandbox.io/s/zen-taussig-3dwph</a>

### Other comments
Looks like type definition is missing.
Using this as workaround, in shims-vue.d.ts:

    declare module "vuetify/lib/services/goto" {
      import Vue from "vue";
      export default function goTo<T extends string | number | HTMLElement | Vue>(target: T, options?: any): any;
    }

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
fixes #9648

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/types/services/goto.d.ts [hunk 1]
```diff
@@ -26,3 +26,5 @@ export interface GoToOptions {
   easing?: VuetifyGoToEasing
   appOffset?: boolean
 }
+
+export default function goTo(target: VuetifyGoToTarget, options?: GoToOptions): Promise<number>
```
Comment (KaelWD) [inline packages/vuetify/types/services/goto.d.ts:30]: This has a different signature in types/index.d.ts

**Review by KaelWD — CHANGES_REQUESTED** [review]

**Comment by KaelWD** [comment]
What's the difference between `maps/vuetify` and `maps/$vuetify`?

File: packages/vuetify/types/services/goto.d.ts (hunk 1 above)
Comment (jacekkarczmarczyk) [reply packages/vuetify/types/services/goto.d.ts:30]: No idea tbh, just updated the signature in both but I guess only one is used

**Comment by jacekkarczmarczyk** [comment]
> What's the difference between maps/vuetify and maps/$vuetify?

No idea, just updated signature in both but I guess only one is used

File: packages/vuetify/types/services/goto.d.ts (hunk 1 above)
Comment (jacekkarczmarczyk) [reply packages/vuetify/types/services/goto.d.ts:30]: It was wrong signature there, fixed

**Review by KaelWD — APPROVED** [review]

**Pull request merged**